manifests/pnp4nagios/service.pp

```diff
@@ -0,0 +1,14 @@
+class icinga2::pnp4nagios::service inherits icinga2::pnp4nagios::params {
+
+  if $service_manage == true {
+    service {$service_name:
```

Comment:
yes, the  $service_name comes from params.pp
if the user change the service name in pnp4nagios class, No, the change in service name doesn't get reflected here